addon/loaders/css.js

```diff
@@ -20,7 +20,12 @@ export default nodeLoader(function css(uri) {
     }
 
     // Try using the default onload/onerror handlers...
-    const link = createLoadElement('link', resolve, reject);
+    const link = createLoadElement('link', resolve, function() {
+      if (this.parentNode) {
+        this.parentNode.removeChild(this);
+      }
+      reject(...arguments);
```

Comment:
I think we can only reject with a single value (its part of the A+ Promise spec I believe), so you can probably just do:

```js
function(error) {
  if (this.parentNode) {
    this.parentNode.removeChild(this);
  }
  reject(error);
}
```